packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -118,26 +133,134 @@ fillDate(
 
 >- `selectCheckbox()` - Enzyme helper that fires a change event with a value for a checkbox at the given name
 
+## Example unit tests
+
+VSP provides helper utilities to make writing tests easier. The following example unit tests illustrate some of these helpers. You can find these unit tests at `src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx`:
```

Comment:
There is no `src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx` in this repo (`veteran-facing-services-tools`). 

Here is the location of that file in the `vets-website` repo: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vic-v2/tests/config/veteranInformation.unit.spec.jsx

Reply:
I made that change. Thanks for finding!